defaulting-latest-build-tools.rst

```diff
@@ -4,31 +4,75 @@
    :location: BCN
    :category: Changelog
 
-Enabling latest versions for Sphinx & Mkdocs builds
-===================================================
+Changes to default project dependencies
+=======================================
 
-We are announcing the deprecation of building with older documentation tool versions by default.
+.. note::
 
-Historically, Read the Docs installed a specific version based on the date your project was created.
+   This post was updated on Oct 10 to reflect all of the changes to installed dependencies
+
+We are announcing the deprecation of automatic installation of several key project dependencies,
+and builds will no longer install older documentation tool versions by default.
+
+Historically, Read the Docs installed a specific version of several dependencies based on the date your project was created.
 This caused odd issues where reimporting a project could change behavior, and caused users to continue using very old versions of build tools that weren't supported by their authors.
 This was done to maintain backwards compatibility,
 but our platform now has robust support for defining a build environment,
 so we encourage you to pin your dependencies instead.
 
-**Starting on October 3rd Read the Docs will install the latest available version of MkDocs and Sphinx by default**.
-We are also not going to be installing other build dependencies, like ``mock`` and ``recommonmark``.
-If you want to use a different version, you can do this by pinning your dependencies.
+**Starting on October 3rd default dependencies installed for each project build will change.**
+If your project was previously depending on these default dependencies,
+your project's builds may encounter errors or failures due to missing packages.
+
+To resolve these build failures we recommend defining all of your project's dependencies.
+See the examples below for guidance on how to update your project configuration
+or learn more about managing dependencies in our `guide for reproducible builds <https://docs.readthedocs.io/en/stable/guides/reproducible-builds.html>`_.
+
+The changes to default dependencies installed during the build process are:
+
+.. list-table::
+   :header-rows: 1
+   :width: 100%
+   :align: center
+   :widths: 10, 5, 5
+
+   - * Package
+     * Prior version
+     * New version
+   - * Sphinx
+     * ``1.8.5``
+     * *Latest release*
+   - * MkDocs
+     * ``0.17.3``
+     * *Latest release*
+   - * ``sphinx-rtd-theme``
+     * *Latest release*
+     * *Not installed*
+   - * ``recommonmark``
+     * *Latest release*
+     * *Not installed*
+   - * ``commonmark``
+     * *Latest release*
+     * *Not installed*
+   - * ``pillow``
+     * *Latest release*
+     * *Not installed*
+   - * ``mock``
+     * *Latest release*
+     * *Not installed*
```

Comment:
Feel free to correct this, I just felt that an explicit mapping helped explain this the most.